be/src/exec/pipeline/exchange/local_exchange_source_operator.cpp

```diff
@@ -75,7 +76,7 @@ Status LocalExchangeSourceOperator::set_finished(RuntimeState* state) {
     // clear _partition_chunk_queue
     { [[maybe_unused]] typeof(_partition_chunk_queue) tmp = std::move(_partition_chunk_queue); }
     // Subtract the number of rows of buffered chunks from row_count of _memory_manager and make it unblocked.
-    _memory_manager->update_row_count(-(full_rows_num + _partition_rows_num));
+    _memory_manager->update_memory_usage(-_memory_manager->get_memory_usage());
```

Comment:
Should only decrease the memory usage of this `LocalExchangeSourceOperator`, but `_memory_manager->get_memory_usage()` is the memory of all the `LocalExchangeSourceOperator`s.

Reply:
my mistake, fixed by introducing  `_local_memory_usage`.